WeScanTests/VisionRectangleDetectorTests.swift

```diff
@@ -66,7 +66,7 @@ final class VisionRectangleDetectorTests: FBSnapshotTestCase {
         quadView.backgroundColor = UIColor.red
         resultView.addSubview(quadView)
 
-        self.FBSnapshotVerifyView(resultView)
+        self.FBSnapshotVerifyView(resultView, perPixelTolerance: 6/256, overallTolerance: 0.02)
```

Comment:
<!--
  0 failure: 
  1 warning:  Operators should ...
  
  
  DangerID: danger-id-Danger;
  File: WeScanTests/VisionRectangleDetectorTests.swift;
  Line: 69;
-->

- :warning: Operators should be surrounded by a single whitespace when they are being used. (`operator_usage_whitespace`)